server/firebase.js

```diff
@@ -30,4 +30,19 @@ firebaseRequests.read = async function(table, uid) {
     return result;
 };
 
+firebaseRequests.verifyIdToken = async function(idToken) {
+  return await auth.verifyIdToken(idToken);
+    // .then(function(decodedToken) {
+    //   console.log("Decoded token: ", decodedToken);
+    //   return true;
+    //   // var uid = decodedToken.uid;
```

Comment:
@mihoward21 you can reconfigure this area to grab info from the token; docs: https://firebase.google.com/docs/reference/admin/node/admin.auth.DecodedIdToken